cmake/BuildLibElf.cmake

```diff
@@ -0,0 +1,15 @@
+include(ExternalProject)
+
+ExternalProject_Add(libelf
+  PREFIX "vendor/elfutils"
+  URL http://archive.ubuntu.com/ubuntu/pool/main/e/elfutils/elfutils_0.163.orig.tar.bz2
```

Comment:
This split makes sense. Let's keep the .so dependency, and let static be static.